stdlib/public/Concurrency/Task.swift

```diff
@@ -596,14 +609,22 @@ public func withUnsafeCurrentTask<T>(body: (UnsafeCurrentTask?) throws -> T) ret
   return try body(UnsafeCurrentTask(_task))
 }
 
-/// An *unsafe* 'current' task handle.
+/// An unsafe task handle for the current task.
 ///
-/// An `UnsafeCurrentTask` should not be stored for "later" access.
+/// To get an instance of `UnsafeCurrentTask` for the current task,
+/// call the `withUnsafeCurrentTask(body:)` method.
+/// Don't try to store an unsafe task handle
```

Comment:
maybe just 
```suggestion
/// Don't store an unsafe task handle
```

?

Reply:
Maybe this phrasing doesn't work as well in reference here.  In TSPL, we use "don't try to do X" to describe something that won't work.  Here, you think you can store an unsafe task handle, but you really can't store it.  I think the counterpoint to my argument is that you *can* store it — just that storing the handle doesn't let you do anything useful.

Reply:
storing it is actively unsafe and would lead to undefined behavior, so the stronger wording would be good hm

Reply:
Removed "try to" in commit 72322eb28db.